tests/drivers/dac/dac_loopback/src/test_dac.c

```diff
@@ -133,6 +133,24 @@
 #define ADC_CHANNEL_ID		1
 #define ADC_1ST_CHANNEL_INPUT	NRF_SAADC_INPUT_AIN1
 
+#elif defined(CONFIG_BOARD_BL5340_DVK_CPUAPP)
+#include <hal/nrf_saadc.h>
+ /* DAC output from MCP4725 pin 1
+  * ADC_2 input read from pin SIO_6
+  */
+
+#define DAC_DEVICE_NAME		DT_LABEL(DT_NODELABEL(dac0))
+#define DAC_RESOLUTION		12
+#define DAC_CHANNEL_ID		0
+
+#define ADC_DEVICE_NAME		DT_LABEL(DT_NODELABEL(adc))
+#define ADC_RESOLUTION		12
+#define ADC_GAIN		ADC_GAIN_1_4
+#define ADC_REFERENCE		ADC_REF_VDD_1_4
+#define ADC_ACQUISITION_TIME	ADC_ACQ_TIME_DEFAULT
+#define ADC_CHANNEL_ID		1
+#define ADC_1ST_CHANNEL_INPUT	NRF_SAADC_INPUT_AIN2
```

Comment:
Can't this be `NRF_SAADC_INPUT_AIN1` as above, so that we don't need a completely new board section? It needs an external connection with a wire anyway, correct? Or is this already wired on the PCB?

Reply:
Just gave it a quick test and it can (it does use an external connection). The fix has been made.